### DISCUSSION TITLE
Support on Edge 840

### DISCUSSION THREAD

**Discussion by Petucky** [house-of-abbey/GarminHomeAssistant#144] [General]
Hi, @philipabbey, I just upgraded my Edge from 830 to 840 and sadly found out this great app is not supported.
Is there anything I can help with to fix the issues?

Thanks,
Peter

**Comment by philipabbey** [comment]
Please see known issues at
https://github.com/house-of-abbey/GarminHomeAssistant?tab=readme-ov-file#known-issues
.

Thank you for your interest.

Philip

On Fri, 14 Jun 2024, 20:33 Petucky, ***@***.***> wrote:

> Hi, @philipabbey <https://github.com/philipabbey>, I just upgraded my
> Edge from 830 to 840 and sadly found out this great app is not supported.
> Is there anything I can help with to fix the issues?
>
> Thanks,
> Peter
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/house-of-abbey/GarminHomeAssistant/discussions/144>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF36JH4PLX3BSXKLEB6FWFDZHNAOVAVCNFSM6AAAAABJK4WNVWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHAZDCMZSGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Philip Abbey

**Comment by Petucky** [comment]
Hi,

Thanks for quick response. I've seen this information, that's why I am asking if there any chance I could help with real device (maybe this was fixed in firmware version released in the meantime or so).

Thanks,
Peter
________________________________

**Comment by philipabbey** [comment]
Do you know how to sideboard an application on your device? I can probably
find the instructions on the 'net.

Sounds like a project for this weekend! I'll be in touch.

Thank you,

Philip

-- 
Philip Abbey

**Comment by Petucky** [comment]
Good hint, fortunately I had a backup of Edge 830, so just copied the binary file and the settings file to 840 and voila - everything works just great! :)
________________________________

**Comment by philipabbey** [comment]
https://github.com/house-of-abbey/GarminHomeAssistant/issues/145

I'll try and get that done today.

On Fri, 14 Jun 2024 at 21:49, Petucky ***@***.***> wrote:

> Good hint, fortunately I had a backup of Edge 830, so just copied the
> binary file and the settings file to 840 and voila - everything works just
> great! :)
> ________________________________
> From: Philip Abbey ***@***.***>
> Sent: 14 June 2024 22:33
> To: house-of-abbey/GarminHomeAssistant ***@***.***>
> Cc: Petucky ***@***.***>; Author ***@***.***>
> Subject: Re: [house-of-abbey/GarminHomeAssistant] Support on Edge 840
> (Discussion #144)
>
>
> Do you know how to sideboard an application on your device? I can probably
> find the instructions on the 'net.
>
> Sounds like a project for this weekend! I'll be in touch.
>
> Thank you,
>
> Philip
>
> --
> Philip Abbey
>
>
> On Fri, 14 Jun 2024, 21:12 Petucky, ***@***.***> wrote:
>
> > Hi,
> >
> > Thanks for quick response. I've seen this information, that's why I am
> > asking if there any chance I could help with real device (maybe this was
> > fixed in firmware version released in the meantime or so).
> >
> > Thanks,
> > Peter
> > ________________________________
> > From: Philip Abbey ***@***.***>
> > Sent: 14 June 2024 21:40
> > To: house-of-abbey/GarminHomeAssistant ***@***.***>
> > Cc: Petucky ***@***.***>; Author ***@***.***>
> > Subject: Re: [house-of-abbey/GarminHomeAssistant] Support on Edge 840
> > (Discussion #144)
> >
> >
> > Please see known issues at
> >
> >
> https://github.com/house-of-abbey/GarminHomeAssistant?tab=readme-ov-file#known-issues
> > .
> >
> > Thank you for your interest.
> >
> > Philip
> >
> > On Fri, 14 Jun 2024, 20:33 Petucky, ***@***.***> wrote:
> >
> > > Hi, @philipabbey <https://github.com/philipabbey>, I just upgraded my
> > > Edge from 830 to 840 and sadly found out this great app is not
> > supported.
> > > Is there anything I can help with to fix the issues?
> > >
> > > Thanks,
> > > Peter
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/house-of-abbey/GarminHomeAssistant/discussions/144>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AF36JH4PLX3BSXKLEB6FWFDZHNAOVAVCNFSM6AAAAABJK4WNVWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWHAZDCMZSGQ>
>
> >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> > --
> > Philip Abbey
> >
> > —
> > Reply to this email directly, view it on GitHub<
> >
> https://github.com/house-of-abbey/GarminHomeAssistant/discussions/144#discussioncomment-9778085>,
>
> > or unsubscribe<
> >
> https://github.com/notifications/unsubscribe-auth/AVUGGYPAHPVHYEFSOJZ6V7DZHNBK3AVCNFSM6AAAAABJK4WNVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TONZYGA4DK>.
>
> >
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/house-of-abbey/GarminHomeAssistant/discussions/144#discussioncomment-9778277>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AF36JH2MEU3EBEHN7RW7NOLZHNFDPAVCNFSM6AAAAABJK4WNVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TONZYGI3TO>
>
> > .
> > You are receiving this because you were mentioned.Message ID:
> >
> <house-of-abbey/GarminHomeAssistant/repo-discussions/144/comments/9778277@
> > github.com>
> >
>
> —
> Reply to this email directly, view it on GitHub<
> https://github.com/house-of-abbey/GarminHomeAssistant/discussions/144#discussioncomment-9778427>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AVUGGYMOB5WF72HBVXOCS3TZHNHQLAVCNFSM6AAAAABJK4WNVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TONZYGQZDO>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/house-of-abbey/GarminHomeAssistant/discussions/144#discussioncomment-9778532>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF36JH4VI4C7JL2U2M33WYTZHNJODAVCNFSM6AAAAABJK4WNVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TONZYGUZTE>
> .
> You are receiving this because you were mentioned.Message ID:
> <house-of-abbey/GarminHomeAssistant/repo-discussions/144/comments/9778532@
> github.com>
>


-- 
Philip Abbey

**Reply by philipabbey** [reply]
Closing this discussion as the devices are now in support.
